internal/cli/auth/login.go

```diff
@@ -110,10 +117,67 @@ Your code will expire after %.0f minutes.
 	opts.AuthToken = accessToken.AccessToken
 	opts.RefreshToken = accessToken.RefreshToken
 	opts.SetUpAccess()
+	_, _ = fmt.Fprintf(opts.OutWriter, "Successfully logged in.\n")
+	if opts.noProfile {
+		return opts.config.Save()
+	}
+	if err := opts.InitStore(ctx); err != nil {
+		return err
+	}
+	_, _ = fmt.Fprintf(opts.OutWriter, "Press Enter to continue your profile configuration")
+	_, _ = fmt.Scanln()
+	if err := opts.askOrg(); err != nil {
+		return err
+	}
+	if err := opts.askProject(); err != nil {
+		return err
+	}
+
+	opts.SetUpProject()
+	opts.SetUpOrg()
+	if err := survey.Ask(opts.DefaultQuestions(), opts); err != nil {
+		return err
+	}
+	opts.SetUpOutput()
+	opts.SetUpMongoSHPath()
 	if err := opts.config.Save(); err != nil {
 		return err
 	}
-	_, _ = fmt.Fprintf(opts.OutWriter, "Successfully logged in\n")
+	_, _ = fmt.Fprintf(opts.OutWriter, "\nYour profile is now configured.\n")
```

Comment:
[q] I'm wondering if given that we are printing so many things here in this package, do we want to maybe create a function that simplifies this output with a void function like `print( "\nYour profile is now configured.\n")` 

and 
```
(opts *loginOpts) print(txt string) {
	_, _ = fmt.Fprintf(opts.OutWriter, txt)
}
```

or is this not very go-ish?

Reply:
not sure if not goish but I gave it a try based on your comment, I decided against it as linters and the IDE loose the ability to flag when you make a formatting error

I intentionally removed one of the params and I no longer get warnings for it

![Screenshot 2022-02-02 at 12 14 23](https://user-images.githubusercontent.com/461027/152151916-2e79d2d0-57f4-4354-ac1f-aabb4968728d.png)

also given the varadic nature of printf wrapping it adds a bit of memory overhead to pack and unpack the args

Reply:
nice check! fair enough